helm-charts/submarine/templates/seldon-secret.yaml

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
I've checked this file and found that we need to replace namespace:
```yaml
namespace: {{ .Release.Namespace }}
```
And please ensure that the license information starts on the first line.

Reply:
I have since discovered that deployments based on this file currently have another problem that needs to be solved (https://issues.apache.org/jira/browse/SUBMARINE-1332). I would therefore suggest that we can remove this file for the time being.

Reply:
ok, I got it.